packages/edit-site/src/components/global-styles/screen-variations.js

```diff
@@ -15,7 +15,8 @@ import ScreenHeader from './header';
 import BlockPreviewPanel from './block-preview-panel';
 import Subtitle from './subtitle';
 
-export function ScreenVariations( { name, path = '' } ) {
+export function ScreenVariations( props ) {
+	const { blockName, name, path } = props;
```

Comment:
Let's keep the destructuring as it was and definitely keep the default value for `path`. Also the new prop is the block's `title` and not a `blockName`.

Reply:
In VariationPanel blockName is used, so I thought this would be more consistent. Will change it.

<img width="336" alt="Screenshot 2023-02-23 at 5 01 24 PM" src="https://user-images.githubusercontent.com/118618474/220962174-2c6a150f-3094-435e-9274-a1b5d74897f4.png">

Reply:
Maybe `blockTitle`? I'm still considering if it's something to pass on `Context` or if it would be better to get it where we needed.

You're too fast applying the changes 😆

Reply:
Haha I have to go now, will be back in an hour to make any more changes if needed!

Reply:
Okay, my suggestion would be to leave `Context` as is with the reasoning it seems to be focused more on routing. So I'd say let's just get the block title inside `ScreenVariations`. We should also use the selector, since it's available.

```
const blockTitle = useSelect(
	( select ) => select( blocksStore ).getBlockType( name ).title,
	[ name ]
);
```

Reply:
I agree with Nik. Let's use `useSelect` and avoid introducing a new prop. That's one of the reasons we have selectors 😄

Reply:
I also agree, just didn't think of it. Still getting used to React and stores 😄 Thanks for the good suggestions and patience, learning a lot here.